### PR TITLE
Redis PUBSUB channel name specification

### PR DESCRIPTION
**Pull request by whatstherub**
To support the operation of many arthur instances spanning projects in a single redis instance.

My python is not super pro, open to any change requests or style improvements.

Specs are green, added some small additional coverage.  Currently have this running in a staged environment.

### CODE REVIEW

File: arthur/worker.py [hunk 1]
```diff
@@ -35,10 +35,22 @@
...
+
+    @pubsub_channel.setter
+    def pubsub_channel(self, value):
+        self.__pubsub_channel = value
+
     def prepare_job_execution(self, job):
         # Fixes the error #479 of RQ. Remove it as soon as it gets fixed.
         # (https://github.com/nvie/rq/issues/479)
-        rq.push_connection(self.connection)
+        # rq.push_connection(self.connection)
```
Comment (whatstherub) [inline arthur/worker.py]: This issue seems indicated to be fixed upstream.  Have not observed any operational issues so far.

**Comment by whatstherub** [comment]
Ignored some async/await flakes:

https://travis-ci.org/chaoss/grimoirelab-kingarthur/builds/482743501

File: arthur/worker.py [hunk 2]
```diff
@@ -60,10 +72,10 @@ def perform_job(self, job, queue):
         }
 
         msg = pickle.dumps(data)
-        self.connection.publish(CH_PUBSUB, msg)
+        self.connection.publish(self.pubsub_channel, msg)
 
         # Fixes the error #479 of RQ. Remove it as soon as it gets fixed.
         # (https://github.com/nvie/rq/issues/479)
-        rq.pop_connection()
+        # rq.pop_connection()
```
Comment (sduenas) [inline arthur/worker.py]: Same here.

File: arthur/worker.py (hunk 1 above)
Comment (sduenas) [reply arthur/worker.py]: I think this can be safely removed. I haven't seen that error in a while.

Comment (whatstherub) [reply arthur/worker.py]: Removed.

File: arthur/worker.py (hunk 2 above)
Comment (whatstherub) [reply arthur/worker.py]: Removed.

**Comment by whatstherub** [comment]
@sduenas I rewrote two of the smaller commit messages to follow what I think you were referring to with he [section] prefixes.  My first squashed commit seems to cover 2 or 3 of these.  I am not sure if [arthur] would be appropriate or not in that case.  If you don't mind rewriting as you see fit and letting me know what you did I can follow moving forward.  It seems like in the future you would prefer more granular commits such that their component focus can be called out.  Sorry for the inconvenience I did not notice the pattern.  I now realize you probably utilize these in your coordinated release notes.

Thank you for the quick response.

**Comment by sduenas** [comment]
@whatstherub Thanks for your fixing the commits. I made some reword as you requested and I pushed the changes to your branch. I don't usually do it. I thought this time was worth because you wanted to saw how I usually write commit messages.

We try to follow (more or less) this [rules]( https://chris.beams.io/posts/git-commit/). As you saw, we usu tags in between brackets. `arthur` tag was fine for the first commits because the commit touches many files.

I think we can merge this PR. Thanks a lot for your contribution.

**Comment by whatstherub** [comment]
OK, I will follow that as a guide in the future.  Let me know if you need anything else before merge.

General question for you, how do you decide when to cut coordinated releases, is there a general cadence established?

**Comment by sduenas** [comment]
> General question for you, how do you decide when to cut coordinated releases, is there a general cadence established?

We usually try to have a new release once per week but we have been busy lately.

**Comment by whatstherub** [comment]
Ok!  Thanks for the assistance.

**Pull request merged**